tools/image_processing/imagej2/imagej2_convert_format/imagej2_convert_format.xml

```diff
@@ -36,24 +36,22 @@
     </outputs>
     <tests>
         <test>
-            <param name="input" value="dot_blot.jpg" ftype="jpg" />
-            <param name="max_heap_size_type" value="megabytes" />
-            <param name="max_heap_size" value="256" />
+            <param name="input" value="dot_blot.jpg" />
             <param name="input_datatype" value="jpg" />
             <param name="output_datatype" value="png" />
-            <output name="output" file="dot_blot.png" ftype="png" />
```

Comment:
Have you tried `compare="sim_size"` instead of diff? I'm not aware of any filetype specific bug. But this does not mean anything :)